setup.py

```diff
@@ -7,21 +7,11 @@
 
 project_base_url = 'https://github.com/LostFan123/pode/'
 
-install_requires = [
-    'lz>=0.8.1',
-    'matplotlib>=2.2.2',
-    'networkx>=2.4',
-    'numpy>=1.16.3',
-    'shapely==1.6.4.post2'
-]
 setup_requires = [
     'pytest-runner>=4.2',
 ]
```

Comment:
this should be removed because user/developer may think that tests invocation using
```bash
> python setup.py test
```
is a valid option when it should be not, because we are doing `import pode` above and it will cause loss in coverage during tests run (not a great thing)